tests/jballerina-unit-test/src/test/resources/test-src/object/object_field_negative.bal

```diff
@@ -24,3 +24,15 @@ class Person {
         self.'error = er;
     }
 }
+
+function testFieldInitializationUsingAnotherField() {
```

Comment:
Is this the correct file for this test? We can add a new file if there's no appropriate object constructor file.

Reply:
Let's add negative tests at module-level also.

Reply:
Removed